cmd/ls/ls.go

```diff
@@ -43,12 +45,30 @@ func formatFileName(name string) string {
 	return name
 }
 
+func lookupIds(uid, gid string) (string, string) {
+	u, err := user.LookupId(uid)
+	if err != nil {
+		fatal(err)
+	}
+	g, err := user.LookupGroupId(gid)
+	if err != nil {
+		fatal(err)
+	}
+	return u.Username, g.Name
+}
+
 func printFileList(fileInfo os.FileInfo) string {
+	stat := fileInfo.Sys().(*syscall.Stat_t)
```

Comment:
@tiago4orion this kind of logic is coupling this implementation on posix or linux ? Perhaps we will need similar to what we done on nash exit, or I'm tripping? :-)